clients/python/rostam/_vecwire.py

```
@@ -0,0 +1,306 @@
+    out += struct.pack(">Q", id)
+    out += _f32be(vec)
+    if flags & _F_TTL:
+        out += struct.pack(">Q", ttl_ms)
+    if has_meta:
+        mj = json.dumps(meta_obj, separators=(",", ":")).encode("utf-8")
+        out += struct.pack(">I", len(mj)) + mj
+    if sparse:
+        idx = list(sparse["indices"])
+        val = list(sparse["values"])
+        out += struct.pack(">I", len(idx))
+        for i, v in zip(idx, val):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)